Nos/Views/NoteOptionsButton.swift

```diff
@@ -27,8 +27,11 @@ struct NoteOptionsButton: View {
             Button {
                 showingOptions = true
             } label: {
-                Image(systemName: "ellipsis")
-                    .foregroundColor(.nosSecondary)
+                VStack {
```

Comment:
There shouldn't be a need to add a VStack here. Just adding `.frame(minWidth: 44, minHeight: 44)` to `Image` seems to do the work in my tests.